<!--
Version:  2017.07.31
Authors:  Members of the Team NAIST-Panasonic at the Amazon Robotics Challenge 2017
          Gustavo A. Garcia R. garcia-g at is.naist.jp (Captain), 
          Lotfi El Hafi, Felix von Drigalski, Wataru Yamazaki, Viktor Hoerig, Arnaud Delmotte, 
          Akishige Yuguchi, Marcus Gall, Chika Shiogama, Kenta Toyoshima, Pedro Uriguen, 
          Rodrigo Elizalde, Masaki Yamamoto, Yasunao Okazaki, Kazuo Inoue, Katsuhiko Asai, 
          Ryutaro Futakuchi, Seigo Okada, Yusuke Kato, and Pin-Chu Yang
*******************
Copyright 2017 Team NAIST-Panasonic 
Licensed under the Apache License, Version 2.0 (the "License");
you may not use this file except in compliance with the License.
You may obtain a copy of the License at 
    http://www.apache.org/licenses/LICENSE-2.0 
Unless required by applicable law or agreed to in writing, software
distributed under the License is distributed on an "AS IS" BASIS,
WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
See the License for the specific language governing permissions and
limitations under the License.
*******************
-->

<!-- Example of how to use -->
<!-- roslaunch tnp_rec_space tnp_rec_space_multiple_SR300.launch camera_1_serial_no:=620202003712 camera_2_serial_no:=617205002089 camera_3_serial_no:="61720300586" camera_4_serial_no:=553201000655 -->

<!-- Launch file for using multiple RealSense R200 cameras -->
<launch>

  <node name="tnp_rec_space" pkg="tnp_recognition_space" type="tnp_recognition_space" output="screen" />
  <node name="tnp_cloud_matching" pkg="tnp_feat_vision" type="tnp_feat_vision_cloud_matching" output="screen" />
  <node name="tnp_feat_vision_record_rgbd_frames" pkg="tnp_feat_vision" type="tnp_feat_vision_record_rgbd_frames" output="screen" />
  <node name="tnp_color_histogram" pkg="tnp_feat_vision" type="tnp_feat_vision_color_histo" output="screen" />
  <node name="tnp_svm" pkg="tnp_svm" type="hog_svm_recognition_node.py" output="screen"/> -->
</launch>
